pkg/kotsadmsnapshot/backup.go

```diff
@@ -36,7 +35,18 @@ import (
 	"k8s.io/apimachinery/pkg/fields"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/client-go/kubernetes"
-	kbclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"k8s.io/utils/ptr"
+	ctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	InstanceBackupNameAnnotation      = "replicated.com/backup-name"
+	InstanceBackupTypeAnnotation      = "replicated.com/backup-type"
+	InstanceBackupsExpectedAnnotation = "replicated.com/backups-expected"
+
+	InstanceBackupTypeKotsadm  = "kotsadm"
+	InstanceBackupTypeApp      = "app"
+	InstanceBackupTypeCombined = "combined"
```

Comment:
should this be called `legacy`? at first i thought `combined` is the over-arching backup type that includes the two separate velero backups. then i realized this is the existing / old backup type after reading the code.

Reply:
it especially made this part confusing to me:
```go
	if hasAppSpec {
		veleroBackup.Annotations[InstanceBackupTypeAnnotation] = InstanceBackupTypeKotsadm
	} else {
		veleroBackup.Annotations[InstanceBackupTypeAnnotation] = InstanceBackupTypeCombined
	}
```

Reply:
renamed